2024-05-06 23:09:40,851 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 23:09:40,851 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Configure stats pid to 7693
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 23:09:40,852 INFO    MainThread:7693 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_230940-p57f0pwq/logs/debug.log
2024-05-06 23:09:40,853 INFO    MainThread:7693 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_230940-p57f0pwq/logs/debug-internal.log
2024-05-06 23:09:40,853 INFO    MainThread:7693 [wandb_init.py:init():561] calling init triggers
2024-05-06 23:09:40,853 INFO    MainThread:7693 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 23:09:40,853 INFO    MainThread:7693 [wandb_init.py:init():586] re-initializing run, found existing run on stack: 7t9vymn4
2024-05-06 23:09:40,857 INFO    MainThread:7693 [wandb_run.py:_finish():2064] finishing run jacobcd52/sae/7t9vymn4
2024-05-06 23:09:40,857 INFO    MainThread:7693 [jupyter.py:save_history():445] not saving jupyter history
2024-05-06 23:09:40,857 INFO    MainThread:7693 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:09:40,857 INFO    MainThread:7693 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-05-06 23:09:40,858 INFO    MainThread:7693 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-06 23:09:40,858 INFO    MainThread:7693 [wandb_run.py:_restore():2287] restore
2024-05-06 23:09:40,858 INFO    MainThread:7693 [wandb_run.py:_restore():2293] restore done
2024-05-06 23:09:45,794 INFO    MainThread:7693 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-05-06 23:09:45,795 INFO    MainThread:7693 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-05-06 23:09:45,808 INFO    MainThread:7693 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-05-06 23:09:46,018 INFO    MainThread:7693 [wandb_init.py:init():611] starting backend
2024-05-06 23:09:46,018 INFO    MainThread:7693 [wandb_init.py:init():615] setting up manager
2024-05-06 23:09:46,022 INFO    MainThread:7693 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-06 23:09:46,023 INFO    MainThread:7693 [wandb_init.py:init():623] backend started and connected
2024-05-06 23:09:46,034 INFO    MainThread:7693 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-05-06 23:09:46,034 INFO    MainThread:7693 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-06 23:09:46,034 INFO    MainThread:7693 [wandb_init.py:init():715] updated telemetry
2024-05-06 23:09:46,040 INFO    MainThread:7693 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-06 23:09:47,095 INFO    MainThread:7693 [wandb_run.py:_on_init():2357] communicating current version
2024-05-06 23:09:47,233 INFO    MainThread:7693 [wandb_run.py:_on_init():2366] got version response 
2024-05-06 23:09:47,233 INFO    MainThread:7693 [wandb_init.py:init():799] starting run threads in backend
2024-05-06 23:09:47,515 INFO    MainThread:7693 [wandb_run.py:_console_start():2335] atexit reg
2024-05-06 23:09:47,516 INFO    MainThread:7693 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-05-06 23:09:47,516 INFO    MainThread:7693 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-05-06 23:09:47,516 INFO    MainThread:7693 [wandb_run.py:_redirect():2280] Redirects installed.
2024-05-06 23:09:47,516 INFO    MainThread:7693 [wandb_init.py:init():842] run started, returning control to user process
2024-05-06 23:10:30,361 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Configure stats pid to 7693
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_231030-mutsadkh/logs/debug.log
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_231030-mutsadkh/logs/debug-internal.log
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8458aa0bb0>
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:init():561] calling init triggers
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 23:10:30,362 INFO    MainThread:7693 [wandb_init.py:init():586] re-initializing run, found existing run on stack: p57f0pwq
2024-05-06 23:10:30,364 INFO    MainThread:7693 [wandb_run.py:_finish():2064] finishing run jacobcd52/sae/p57f0pwq
2024-05-06 23:10:30,364 INFO    MainThread:7693 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-06 23:10:30,364 INFO    MainThread:7693 [wandb_run.py:_restore():2287] restore
2024-05-06 23:10:30,365 INFO    MainThread:7693 [wandb_run.py:_restore():2293] restore done
2024-05-06 23:10:35,194 INFO    MainThread:7693 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-05-06 23:10:35,194 INFO    MainThread:7693 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-05-06 23:10:35,200 INFO    MainThread:7693 [wandb_run.py:_footer_sync_info():3895] logging synced files
